### PR TITLE
Fixed some typos in cylc.flow.network module

### LINKED ISSUE #3396: Fix some typos in cylc.flow.network package
**Describe the bug**

It looks like we have a few typos in the `cylc.flow.network` package (e.g. in `schema` we have _componnents_, _formated_, etc). It would be nice to get someone to review the strings in this package and see if there are any other typos.

**Release version(s) and/or repository branch(es) affected?**

`master`, affects 8.0a1 (some are in documentation generated)

**Steps to reproduce the bug**

NA

**Expected behavior**

NA

**Screenshots**

**Additional context**

**Pull requests welcome!**
This is an Open Source project - please consider contributing a bug fix
yourself (please read `CONTRIBUTING.md` before starting any work though).

### PR DESCRIPTION
**Pull request by dveselov**
Hi,

These changes close #3396 

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests (why?).
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.

### CODE REVIEW

**Comment by kinow** [comment]
@dveselov that was quick! And I think spot on! Looks like you found even more typos than I was expecting! Thank you for your contribution.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ requests_).
  - David Sutherland
  - Martin Ryan
  - Tim Pillinger
+ - Dima Veselov
```
Comment (kinow) [inline CONTRIBUTING.md]: Thank you for following the CONTRIBUTING.md instructions! :tada:

**Comment by kinow** [comment]
@dwsutherland can you take a look, just to confirm nothing will break in the GraphQL parts... I don't think anything will break... but just in case. If approved, that will be the second approval, and this PR should be ready to be merged then :+1:

**Comment by dveselov** [comment]
Thank you too, guys.
Happy Hacktoberfest :+1:

**Comment by dwsutherland** [comment]
@dveselov - Looks good, just a conflict in contributor (not sure if it count if I contribute by resolving?)

**Comment by kinow** [comment]
>(not sure if it count if I contribute by resolving?)

Oh, if you can fix that, all good IMO :+1:

**Pull request merged**